examples/cisco/demo_vlan.pp

```diff
@@ -31,4 +31,15 @@
     state           => 'active',
     fabric_control  => $fabric_control
   }
+  # For private vlan
+  if platform_get() =~ /n(3|5|6|7|9)k/ {
+    cisco_vlan { '120':
```

Comment:
This needs to be a different vlan id or it wil conflict with the cisco_vlan 120 resource in interface.pp